packages/react/src/components/NumberInput/NumberInput.tsx

```diff
@@ -138,27 +313,34 @@ const NumberInput = React.forwardRef(function NumberInput(props, forwardRef) {
     }
   }
 
-  const handleFocus = (evt) => {
-    if (evt.target.type === 'button') {
+  const handleFocus: React.FocusEventHandler<
+    HTMLInputElement | HTMLDivElement
+  > = (evt) => {
+    if ('type' in evt.target && evt.target.type === 'button') {
       setIsFocused(false);
     } else {
       setIsFocused(evt.type === 'focus' ? true : false);
     }
   };
 
   const outerElementClasses = cx(`${prefix}--form-item`, {
-    [customClassName]: !!customClassName,
+    [customClassName!]: !!customClassName,
```

Comment:
Same thing here with the eslint disable line rule. It can be scoped to the specific rule

Reply:
Ah thanks for catching! I hadn't noticed that lint passes but with warnings. Now that I have ESLint set up in VSCode I'll catch these much earlier.